MongoDB/services/commentService.js

```diff
@@ -0,0 +1,55 @@
+const Comment = require("../models/Comment");
+
+const fetchCommentData = async (
+    sortBy = "name:ASC",
+    page = 1,
+    itemPerPage = 5
+) => {
+    page = parseInt(page);
+    itemPerPage = parseInt(itemPerPage);
+
+    const arr = sortBy.split(":");
+    let field = arr[0];
+    let order = arr[1] === "ASC" ? 1 : -1;
+
+    if (arr[1] === undefined) order = 1;
+
+    if (Number.isNaN(page)) {
+        const error = new Error("Page number should be in digit");
+        error.status = 400;
+        throw error;
+    }
+
+    if (Number.isNaN(itemPerPage)) {
+        const error = new Error("itemPerPage should be in digit");
+        error.status = 400;
+        throw error;
+    }
+
+    const filter = {};
+    filter[field] = order;
```

Comment:
you can directly assign a field at the file of the declaration. See below code.

`
const filter = {
    [field]: order
}
`